material/src/main/java/org/odk/collect/material/MaterialProgressDialogFragment.java

```diff
@@ -52,29 +62,18 @@ public void setTitle(String title) {
     }
 
     public void setMessage(String message) {
-        setArgument(message, MESSAGE);
+        setArgument(MESSAGE, message);
 
         AlertDialog dialog = (AlertDialog) getDialog();
         if (dialog != null) {
             setupView(dialog);
         }
     }
 
-    /*
-    We keep this just in case to avoid problems if someone tries to show a dialog after
-    the activity’s state have been saved. Basically it shouldn't take place since we should control
-    the activity state if we want to show a dialog (especially after long tasks).
-     */
     @Override
-    public void show(FragmentManager manager, String tag) {
-        try {
-            manager
-                    .beginTransaction()
-                    .add(this, tag)
-                    .commit();
-        } catch (IllegalStateException e) {
-            Timber.w(e);
-        }
+    public void setCancelable(boolean cancelable) {
+        setArgument(CANCELABLE, cancelable);
```

Comment:
Without this, the "cancellable" behaviour wouldn't have been persisted between recreations of the fragment.